source/content/terminus/install.md

```diff
@@ -65,6 +66,14 @@ curl -O https://raw.githubusercontent.com/pantheon-systems/terminus-installer/ma
 
 See [Troubleshooting](#troubleshooting) if your installation fails, or the [Installation](https://github.com/pantheon-systems/terminus#installation) section of the Terminus README file on GitHub for advanced installation methods.
 
+### Homebrew Installation
+
+The Terminus application is published to Homebrew, to install it you should just run:
+
+```bash
+brew install pantheon-systems/external/terminus
```

Comment:
Yeah, we should leave the brew install line as shown above, and update it later after the T3 release, as @kporras07 said.

Reply:
this can be changed (and players at home can run `brew cleanup terminus` to remove the beta bits)

this line should be:

```bash
brew install terminus
```